manuscript/code/hello-world-minified.c

```diff
@@ -1 +1 @@
-uint64_t*foo;uint64_t*main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
\ No newline at end of file
+uint64_t*foo;uint64_t main(){foo="Hello World!    ";while(*foo!=0){write(1,foo,8);foo=foo+1;}}
```

Comment:
@ChristianMoesl The intention here is to have zero whitespace...even if there is a compiler warning...

Reply:
@ckirsch I see. But why do we have to have zero whitespaces? For Teaching? Isn't this a good example for the limitations of minification?

Reply:
@ChristianMoesl To demonstrate that whitespace in C programs has no effect on semantics.

Reply:
@ckirsch Ok i see. I'm closing this.